routes/artist.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express"); 
+const router = express.Router(); 
+const artist = require("../models/newArtist");
+const Artist = require("../models/newArtist");
+
+router.post("/", (req, res) => {
+    let userid = req.body.userid; 
+    let name = req.body.name; 
+    let nationality = req.body.nationality; 
+    let medium = req.body.medium;
+    let artist = new Artist({userid: userid, name: name, nationality: nationality, medium: medium});
```

Comment:
This could be condensed into ```
let artist = new Artist({
   userid: req.body.userid,
   name: req.body.name,
   nationality: req.body.nationality,
   medium: req.body.medium
   });```

Reply:
I'd also use const, since we're not actually changing the user object once it has been created :)